Added italics

SteveMcGrath opened the pull request:
Italics were added to the `keyword`, `identifier.argument`, and `identifier.decorator` definitions to more make them more visually pronounced.  This becomes even more striking when using a font that treats italics as cursive (Operator, Dank Mono, etc.)

onecrayon commented:
Hm, I'll give this some thought and try it out locally; I'm not sure I like the idea of italicizing keywords, in particular, but I admittedly haven't played around with it. In the meantime, can you please adjust the spacing so that it matches the original file? It looks like your editor added a bunch of extraneous changes when you saved it. 🙂

SteveMcGrath commented:
Amusingly, Nova did that to me automatically when i saved.  Here is how it looks with Dank Mono

![Screen Shot 2020-11-05 at 11 14 17 AM](https://user-images.githubusercontent.com/254700/98274090-a0c85a00-1f58-11eb-93ad-18f58983d442.png)

Fonts that dont take advantage of it, wont use it, for example, Monaco:

![Screen Shot 2020-11-05 at 11 19 30 AM](https://user-images.githubusercontent.com/254700/98274322-e422c880-1f58-11eb-9b01-48423940ed9b.png)

SteveMcGrath committed:
Added italics support

italics were added to the following declarations:

keyword
identifier.argument
identifier.decorator
